examples/getting_started/renate_config.py

```diff
@@ -93,7 +93,7 @@ def buffer_transform() -> Callable:
 
 
 def metrics_fn() -> Dict:
-    return {"accuracy": Accuracy()}
+    return {"accuracy": Accuracy(task="multiclass", num_classes=10)}
```

Comment:
We need to be very careful with this new `torchmetrics` interface. It caused me some troubles in another project. By default it uses a `"macro"` average for multiclass accuracy, i.e., it computes average accuracy for each label, then averages those. We probably want the `"micro"` average, right?

Reply:
That is correct but we are using Accuracy which uses "micro": https://torchmetrics.readthedocs.io/en/stable/classification/accuracy.html#accuracy
I've initially used the MulticlassAccuracy and then the tests failed because of the macro averaging. Now they pass, so things should work as intended.

Reply:
ok, I've changed everything to multiclass and added the `average` argument